File: Backend/maintenace-service/src/Entity/Esp.cs
namespace Entity
{
    public class Esp
    {
        public string Id { get; set; }              // Id interno del registro
        public string Nombre { get; set; }          // Nombre 
        public string IdComp { get; set; }          // FK de la tabla Compania
        public bool Estado { get; set; }            // Estado
        public DateTime Fecha_log { get; set; }      // Log fecha
    }
}

File: Frontend/src/Types/esp.ts
export interface Esp {
  id: string;
  nombre: string;
  idComp: string;
  estado: boolean;
  fecha_log: string;
}

export interface EspResponse {
  data: Esp[];
  message: string;
  success: boolean;
}

export interface EspCreateRequest {
  id: string;
  nombre: string;
  idComp: string;
  estado: boolean;
  fecha_log: string;
}

export interface EspUpdateRequest extends EspCreateRequest {
  id: string;
}
